Canary CI: ` Package 'pathrs', required by 'virtual:world', not found`

https://github.com/containerd/nerdctl/actions/runs/23084081217/job/67058199688

```
> [build-runc 4/4] RUN GO=xx-go CC=$(xx-info)-gcc STRIP=$(xx-info)-strip make static &&   xx-verify --static runc && cp -v -a runc /out/runc.amd64:
0.335 xx-go build -trimpath -buildmode=pie  -tags "seccomp urfave_cli_no_docs libpathrs netgo osusergo" -ldflags "-X main.gitCommit=v1.5.0-rc.1-0-g5d2588d  -linkmode external -extldflags -static-pie " -o runc .
36.98 # cyphar.com/go-pathrs/internal/libpathrs
36.98 # [x86_64-linux-gnu-pkg-config --cflags  -- pathrs]
36.98 Package pathrs was not found in the pkg-config search path.
36.98 Perhaps you should add the directory containing `pathrs.pc'
36.98 to the PKG_CONFIG_PATH environment variable
36.98 Package 'pathrs', required by 'virtual:world', not found
60.14 make: *** [Makefile:104: static-bin] Error 1
```

Caused by
- https://github.com/opencontainers/runc/pull/5103

--- a/Dockerfile
+++ b/Dockerfile
@@ -97,7 +97,8 @@ WORKDIR /go/src/github.com/opencontainers/runc
 RUN git-checkout-tag-with-hash.sh ${RUNC_VERSION} && \
   mkdir -p /out
 ENV CGO_ENABLED=1
-RUN GO=xx-go CC=$(xx-info)-gcc STRIP=$(xx-info)-strip make static && \
+# FIXME: avoid omitting libpathrs
+RUN set -x ; GO=xx-go CC=$(xx-info)-gcc STRIP=$(xx-info)-strip make BUILDTAGS="$(grep -oP "^BUILDTAGS := \K.*" Makefile  | sed -e s/libpathrs//)" static && \
   xx-verify --static runc && cp -v -a runc /out/runc.${TARGETARCH}
 
 FROM build-base AS build-bypass4netns

